include/boost/url/url_base.hpp

```diff
@@ -1659,6 +1674,91 @@ class url_base::segments_type
     BOOST_URL_DECL
     iterator
     end() const noexcept;
+
+    /** Erase the specified sequence of path segments.
+
+        @par Exception Safety
+
+        No-throw guarantee.
+
+        @param first The first segment to be erased.
+
+        @param last One past the last segment to be erased.
+    */
+    BOOST_URL_DECL
+    iterator
+    erase( iterator first, iterator last ) noexcept;
+
+    /** Erase the specified path segment
+
+        @param pos The path segment to erase
+
+        This function is equivalent to:
```

Comment:
Telling the user what it is equivalent to in terms of some other function isn't as useful as actually just saying what it does. And you also need to say something about iterator invalidation.